src/main/java/net/datafaker/providers/base/Image.java

```diff
@@ -0,0 +1,97 @@
+package net.datafaker.providers.base;
+
+import net.datafaker.providers.entertainment.EntertainmentProviders;
+
+import javax.imageio.ImageIO;
+import java.awt.Color;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Base64;
+
+import static java.awt.Color.WHITE;
+
+/**
+ * Generates base64 encoded PNG, GIF, JPG and SVG images.
+ *
+ * @since 2.3.0
+ */
+public class Image extends AbstractProvider<EntertainmentProviders> {
+
+    private static final int WIDTH = 256;
+    private static final int HEIGHT = 256;
+    private static final int BOX_SIZE = WIDTH / 8;
```

Comment:
Change the visibility to make it usable in the generator class?

Edit: If the generator class is being kept.

Reply:
generator is gone.